PendingReleaseNotes

```diff
@@ -162,6 +162,9 @@
     fixes and enhancements.
   * For more detailed information see:
     https://docs.ceph.com/en/latest/rados/configuration/mclock-config-ref/
+* CEPHFS: Renaming "mds_kill_skip_replaying_inotable", which is a bit confusing
+    to "mds_kill_after_journal_logs_flushed".
```

Comment:
nit: `MDS config option name "mds_kill_skip_replaying_inotable" is a bit confusing, renaming it to "mds_kill_after_journal_logs_flushed"`

else LGTM

Reply:
Done. thanks.